Assets/PackageStagingArea/Sun/Runtime/Scripts/ToggleSpriteSwap.cs

```diff
@@ -12,12 +12,26 @@ public class ToggleSpriteSwap : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
+        if (toggle == null) GetComponent<Toggle>();
```

Comment:
`toggle = GetComponent<toggle>();`

Reply:
Dit is bewust omdat de Toggle niet altijd op dezelfde game object zit maar in een child kan zitten; als er een specifieke toggle is opgegeven, dan moet juist niet een toggle opgezocht worden

Reply:
ik bedoel dat er nu alleen een GetComponent staat, maar niet het `toggle =` gedeelte, dus je toggle blijft null als die null was

Reply:
🤦 Dank je wel dat je dat nog even aangeeft, helemaal overheen gelezen en: scherp opgemerkt!